src/pages/become-a-sponsor.js

```diff
@@ -104,7 +104,7 @@ export default () => {
           <thead>
             <tr>
               <th>Benefit</th>
-              <th css={styles.highlight}>Platinum (Limit 1)</th>
+              <th css={styles.highlight}>Platinum</th>
```

Comment:
I think leaving limit one here may be better than listing limits on all of them.